alembic/versions/52209719b79e_extend_announcements_with_media_and_.py

```diff
@@ -0,0 +1,38 @@
+"""extend announcements with media and info entries
+
+Revision ID: 52209719b79e
+Revises: 5a633236f075
+Create Date: 2017-08-30 09:48:07.507323
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '52209719b79e'
+down_revision = '5a633236f075'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSON
+
+field = 'media_url'
+
+def make_timestamp():
+    now = datetime.datetime.utcnow()
+    return now.isoformat()
+
+
+def upgrade():
+    op.add_column('announcement', sa.Column(field, sa.String))
+    op.add_column('announcement', sa.Column('info', JSON))
+    op.add_column('announcement', sa.Column('published', sa.Boolean, default=False))
+    op.add_column('announcement', sa.Column('updated', sa.Text,
+                                       default=make_timestamp))
+    sql = 'update announcement set published=true'
```

Comment:
You are missing here media_url

Reply:
it's there, only not good visible. Changed that in https://github.com/Scifabric/pybossa/pull/1644/commits/f2ff63cb6de024e5336587a1bd8ad8c2a12eb32b